### PR TITLE
fix(tests): fix broken tests

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

File: alexandria/core/tests/test_views.py [hunk 1]
```diff
@@ -44,7 +46,7 @@ def test_anonymous_writing(db, document, client, settings, user, allow_anon, met
 
     resp = getattr(client, method)(url, data=data)
     assert (
-        resp.status_code == HTTP_201_CREATED or HTTP_200_OK
+        resp.status_code in {HTTP_201_CREATED, HTTP_200_OK}
```
Comment (fugal-dy) [inline alexandria/core/tests/test_views.py:49]: is there a particular rationale for the usage of `{` ... `}` or just to spice up things a little? ;)

File: alexandria/core/tests/test_views.py [hunk 2]
```diff
@@ -196,23 +199,26 @@ def test_at_rest_encryption(admin_client, settings, document, mocker):
...
     db,
     settings,
+    attribute,
     viewset,
     request,
     update,
     admin_client,
+    admin_user,
+    admin_groups,
     active_group,
```
Comment (fugal-dy) [inline alexandria/core/tests/test_views.py:209]: `active_group` is somewhat misleading after parametrizing the `group` and `user` attribute

rename to `active_attribute`?

File: alexandria/core/tests/test_views.py [hunk 3]
```diff
@@ -247,7 +253,7 @@ def test_validate_created_by_group(
         url = reverse(f"{model_name}-detail", args=[model.pk])
         # updates are not allowed to change created_by_group
         model.refresh_from_db()
-        assert model.created_by_group == "somegroup"
+        assert getattr(model, f"created_by_{attribute}") == "somegroup"
```
Comment (fugal-dy) [inline alexandria/core/tests/test_views.py:256]: does this hold for `attribute` == `user`

File: alexandria/core/tests/test_views.py (hunk 2 above)
Comment (Yelinz) [reply alexandria/core/tests/test_views.py]: Makes sense done

File: alexandria/core/tests/test_views.py (hunk 3 above)
Comment (Yelinz) [reply alexandria/core/tests/test_views.py]: Yes, changed the the name to be something more generic

**Review by fugal-dy — APPROVED** [review]
well done

**Pull request merged**